Reorder and add https://lnnodeinsight.com/ to the discover page with a new category: Node management

SiddheshKukade opened the pull request:
### Describe the changes you have made in this PR
Reordered the sections on the Discover page and added  new section of node management with lnnodeinsight.com as a card.
### Link this PR to an issue [optional]

Fixes   #2111

### Type of change
 

- `refactor`: refactored the sections of the discover page.
- `feat`: New feature (non-breaking change which adds functionality)
 

### Screenshots of the changes [optional]
![image](https://user-images.githubusercontent.com/65951872/226101280-2645ebd2-f559-4067-8885-c7c8f96de9e0.png)
![image](https://user-images.githubusercontent.com/65951872/226103651-275cc39a-dbc8-49de-89ff-61b992582fb4.png)
### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

SiddheshKukade commented:
Please assign someone to review.

reneaaron commented on src/app/screens/Discover/websites.json:
```diff
@@ -162,6 +110,81 @@
+      {
+        "title": "LNnodeinsight.com",
+        "subtitle": "Develop your own data-driven Lightning Network insight",
+        "logo": "https://lnnodeinsight.com/www/LNnodeinsight-tinylogonobkgrnd-white.svg",
```
@MoritzKa Can you please upload the logos files to our CDN and replace the paths here?

escapedcat replied:
Try this:
https://cdn.getalby-assets.com/alby-extension-website-screen/LNnodeinsight-tinylogonobkgrnd.svg

SiddheshKukade replied:
Done, Here's how it looks :
![image](https://user-images.githubusercontent.com/65951872/226714188-4e945314-d9e6-4860-b3f2-e79f5bc6eab4.png)

reneaaron reviewed (commented):
Thanks for the PR @SiddheshKukade, looks good! 🙌 

Will merge as soon as @MoritzKa replaced the images.

SiddheshKukade commented:
My apologies for the extra commits. I accidentally pushed them into the wrong branch. 
@MoritzKa  Sir, the code is ready to be reviewed.

MoritzKa commented:
The UI looks good and links are working. I did not check the code though.

SiddheshKukade commented:
> The UI looks good and links are working. I did not check the code though.

@MoritzKa the changes are mostly some JSON refactoring and added code for Node management and added section for LND nodeinsight.com. Let me know if anything's unclear in code.

reneaaron commented:
Thanks for your work on this PR @SiddheshKukade 🙌

SiddheshKukade commented:
@reneaaron it's my pleasure ☺️
